### ISSUE TITLE
data: URLs not working

### ISSUE THREAD

**Issue by Tibladar** [DarkTrick/htmlfilemerger#12]
htmlfilemerger tries to find the base64 string of a `data:` URL as file instead of just copying it.

Example:
```
<html>
<body>
<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4
  //8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==" alt="Red dot" />
</body>
</html>
```

```
Problems occured
  Error: Line 3: Could not find file `data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4
  //8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==`; searched in `/home/user/data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4
  //8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==`.
```

Could be fixed by checking for `data:` like
```
if not src.startswith("data:"):
        src = self._getEncodedImageContent (src)
```

**Comment by DarkTrick** [comment]
Thank you very much for the report!
I might need a little time to fix this, but I'll do it as quickly as possible.

**Comment by DarkTrick** [comment]
fixed in latest version (commit 0c29d5448e9d46d052d79e8081cd6f788bbdaf20 )
